multinet/user.py

```diff
@@ -137,3 +138,22 @@ def filtered_user(user: User) -> FilteredUser:
 def copy_user(user: User) -> User:
     """Create and return a new instance of User."""
     return from_dict(User, dataclasses.asdict(user))
+
+
+def search_user(query: str) -> Cursor:
+    """Search for users given a partial string."""
+
+    coll = user_collection()
+    aql = read_only_db("_system").aql
+
+    bind_vars = {"@users": coll.name, "query": query}
```

Comment:
What does the `@` do?

Reply:
It's just the required syntax for bind vars https://www.arangodb.com/docs/stable/aql/fundamentals-bind-parameters.html